docs/fsharp/style-guide/conventions.md

```diff
@@ -86,7 +86,7 @@ let parsed = StringTokenization.parse s // Must qualify to use 'parse'
 
 In F#, the order of declarations matters, including with `open` statements. This is unlike C#, where the effect of `using` and `using static` is independent of the ordering of those statements in a file.
 
-In F#, because elements opened into a scope can shadow others already present. This means that reordering `open` statements can alter the meaning of code. As a result, sorting alphanumerically (or pseudorandomly) is generally not recommended, lest you generate different behavior that you might expect.
+In F#, elements opened into a scope can shadow others already present. This means that reordering `open` statements could alter the meaning of code. As a result, sorting all `open` statements alphanumerically (or pseudorandomly) is generally not recommended, lest you generate different behavior that you might expect.
```

Comment:
Why does it make sense to mention pseudorandom sorting of `open` statements here? Is that something someone would actually consider?

Reply:
That's fair, the issue here is _any_ sorting which isn't topological is likely to result in unwanted shadowing.